server/backend/api/app/routers/router_user_role_assignment.py

```diff
@@ -167,3 +171,36 @@ def delete_user_role_assignment(
                 user_role.role.client_number.forest_client_number
             )
         audit_event_log.log_event()
+
+
+def __send_user_access_granted_email(
```

Comment:
In "admin_management", this method probably can be in a service; here in server/backend, I placed it here in router (don't really like it to be in CRUD) as a private method.

Reply:
As discussed, moved send_user_access_granted_email from private method for router into "crud_user_role" file as public method. Let router dispatches to send email as it should.